test/model/V4NodeMetrics.spec.js

```diff
@@ -0,0 +1,73 @@
+ *
+ * OpenAPI spec version: 4.0.0
+ *
+ * NOTE: This class is auto generated by the swagger code generator program.
+ * https://github.com/swagger-api/swagger-codegen.git
+ *
+ * Swagger Codegen version: 2.2.3
+ *
+ * Do not edit the class manually.
+ *
+ */
+
+(function(root, factory) {
+  if (typeof define === 'function' && define.amd) {
+    // AMD.
+    define(['expect.js', '../../src/index'], factory);
+  } else if (typeof module === 'object' && module.exports) {
+    // CommonJS-like environments that support module.exports, like Node.
+    factory(require('expect.js'), require('../../src/index'));
+  } else {
+    // Browser globals (root is window)
+    factory(root.expect, root.GiantSwarmV4);
+  }
+}(this, function(expect, GiantSwarmV4) {
+  'use strict';
+
+  var instance;
+
+  beforeEach(function() {
+    instance = new GiantSwarmV4.V4NodeMetrics();
+  });
+
+  var getProperty = function(object, getter, property) {
+    // Use getter method if present; otherwise, get the property directly.
+    if (typeof object[getter] === 'function')
+      return object[getter]();
+    else
+      return object[property];
+  }
+
+  var setProperty = function(object, setter, property, value) {
+    // Use setter method if present; otherwise, set the property directly.
+    if (typeof object[setter] === 'function')
+      object[setter](value);
+    else
+      object[property] = value;
+  }
+
+  describe('V4NodeMetrics', function() {
+    it('should create an instance of V4NodeMetrics', function() {
+      // uncomment below and update the code to test V4NodeMetrics
+      //var instane = new GiantSwarmV4.V4NodeMetrics();
```

Comment:
Is this generated code? what's the idea here.. Also theres a typo in all the commented examples.

Reply:
It's all generated

Reply:
I actually fixed that already https://github.com/swagger-api/swagger-codegen/commit/90859575ebfea800dde93a498e1439011963d6cf